### PR TITLE
add reminder about plotline installation

### PR DESCRIPTION
**Pull request by naheld**
This addresses issue #426 
Adds a reminder callout which links to the setup page, in case the learner has not yet installed plotnine.

### CODE REVIEW

File: _episodes/07-visualization-ggplot-python.md [hunk 1]
```diff
@@ -30,6 +30,12 @@ and [The Grammar of Graphics][grammar-of-graphics]
 by Leland Wilkinson. The `plotnine`
 package is built on top of Matplotlib and interacts well with Pandas.
 
+> ## Reminder
+> `plotnine` is not included in the standard Anaconda installation and needs
+> to be installed separately. If you haven't done so already, you can find installation
+> instructions on the Setup page: <https://datacarpentry.org/python-ecology-lesson/setup.html#download-plotting-package>.
```
Comment (maxim-belkin) [inline _episodes/07-visualization-ggplot-python.md:36]:
```suggestion
> instructions on the [Setup page][setup-plotnine].
```

Comment (maxim-belkin) [reply _episodes/07-visualization-ggplot-python.md:36]: I suggest creating `lesson-setup-plotnine` link at the end of this document (line 601) like so:

```markdown
[setup-plotnine]: https://datacarpentry.org/python-ecology-lesson/setup.html#download-plotting-package
```

**Comment by maxim-belkin** [comment]
Thank you for the pull request, Noelle! I agree that it is useful to remind people that they need to install `plotnine`. My only suggestions would be to define `setup-plotnine` link and using that in the text.

**Comment by naheld** [comment]
Hi Maxim - thanks! I agree with you. I resolved your change and also added the plotnine link.

**Pull request merged**